revisions/N0/src/text/dialog_4.asm

```diff
@@ -446,10 +445,9 @@ Dialog1BE::
 Dialog1BF::
 Dialog1C0::
     db "Å?! #####, jeg  "
-    db "ser du har en   "
```

Comment:
Litt for lite dialekt for Tarin syntes jeg

Reply:
e9